Gruntfile.js

```diff
@@ -147,18 +244,23 @@ module.exports = function (grunt) {
         },
 
         clean: {
-            doc: ['doc']
+            doc: ['doc'],
+            visualTests: ['visual-tests/dist']
         }
     });
 
     require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks);
+    grunt.loadNpmTasks('assemble');
```

Comment:
I can't spot any difference although my phone is having a nightmare rendering the npm site...